frontend/src/host.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+const CreateCall = () => {
+  const [peerId, setPeerId] = useState('');
+
+  const videoRef = useRef();
+  const audioRef = useRef();
+
+  useEffect(() => {
+    navigator.mediaDevices
+      .getUserMedia({
+        video: true,
+      })
+      .then((stream) => {
+        videoRef.current.srcObject = stream;
+      });
+
+    import('peerjs').then((module) => {
+      const Peer = module.default;
+      const peer = new Peer({
+        host: 'polar-escarpment-94286.herokuapp.com',
```

Comment:
can we avoid this being hard-coded in and set it in an environment properties file